Libraries/test/Amazon.Lambda.RuntimeSupport.Tests/Amazon.Lambda.RuntimeSupport.UnitTests/HandlerTests.cs

```diff
@@ -395,16 +393,20 @@ private async Task<ExecutionInfo> ExecHandlerAsync(string handler, string dataIn
                     Client = testRuntimeApiClient
                 };
 
+                var loggerAction = actionWriter.ToLoggingAction();
```

Comment:
This was moved down because now that the tests are targeting .NET 8 instead of .NET Core 2.1 it uses the newer log level logger. The problem with that is the `new LambdaBootstrap(handlerWrapper, initializer.InitializeAsync)` done a little bit above this overrides the test hack done by `UserCodeLoader.SetCustomerLoggerLogAction` to redirect the logging. I put this after `new LambdaBootstrap(handlerWrapper, initializer.InitializeAsync)` to override custom logger action after bootstrap does its work.